feature-libs/customer-ticketing/components/customer-ticketing-reopen/customer-ticketing-reopen-dialog/customer-ticketing-reopen-dialog.component.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { I18nTestingModule } from '@spartacus/core';
+import { LaunchDialogService } from '@spartacus/storefront';
+
+import { CustomerTicketingReopenDialogComponent } from './customer-ticketing-reopen-dialog.component';
+
+class MockLaunchDialogService implements Partial<LaunchDialogService> {
+  closeDialog(_reason: string): void {}
+}
+
+describe('CustomerTicketingReopenDialogComponent', () => {
+  let component: CustomerTicketingReopenDialogComponent;
+  let fixture: ComponentFixture<CustomerTicketingReopenDialogComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [I18nTestingModule],
+      declarations: [CustomerTicketingReopenDialogComponent],
+      providers: [
+        { provide: LaunchDialogService, useClass: MockLaunchDialogService },
+      ],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CustomerTicketingReopenDialogComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Test for characters lefts and data truncation in case of copy and paste
Test for form validity to force message is not empty